Assignment-1 Submission

paula-aoy opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Submit Assignment-1

## What did you learn from the changes you have made?
It's hard for me to do the live coding during class live, because I can't follow the instructions and code at the same time. But the assignment helps put it into practice and the livecode books also help a lot!

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
No.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
No.

## How were these changes tested?
Running codes.

## A reference to a related issue in your repository (if applicable)
No.

## Checklist
- [X] I can confirm that my changes are working as intended

paula-aoy committed:
Assignment-1 Submission

paula-aoy committed:
Update Assignment 1

paula-aoy committed:
Update 2

KasraVakiloroayaei reviewed (changes requested):
A1: Incomplete 

Great work, however I believe you did not use the standardized set of predictors in your KNN  fit, thus getting the incorrect hyperparameter value for k. Please make this correction, then print the accuracy score.

paula-aoy committed:
Assignment 1 - Update

paula-aoy commented:
thank you for your feedback, @KasraVakiloroayaei 
I see what you mean, so I changed the code on item (iv) when setting up the standardized predictors to train the KNN (wine_df_standardized). Pull request also updated. Would that make it correct? 
all the best!
